### PR TITLE
Create 404.md

### PR DESCRIPTION
**Pull request by paulvi**
2nd option for friendlier 404 page

ref https://github.com/apache/struts-site/pull/48#issuecomment-349885748

The rendering of js within page was not tested yet.

### CODE REVIEW

**Comment by lukaszlenart** [comment]
I think this won't work without an entry in `.htaccess`

**Comment by lukaszlenart** [comment]
👍

**Comment by paulvi** [comment]
yes, it should not work without #73 .htaccess change

File: source/404.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+title: 404 Page Not Found
+---
+
+# 404 Page Not Found
+
+The page was moved or never existed.  
+Use menu above to navigate.  
+You will be redirected to main page in 5 second.
+
+<div onLoad="setTimeout('redirectToHome()', 5000)">
```
Comment (yasserzamani) [inline source/404.md]: I think only `body` has `onLoad` attribute. I think below instead would work:

```diff
-<div onLoad="setTimeout('redirectToHome()', 5000)">
 <script type="text/javascript">
    function redirectToHome() {
        window.location = "http://struts.apache.org/"
    }
+  setTimeout(redirectToHome, 5000);
```

Comment (paulvi) [reply source/404.md]: Thanks.

First, it works nice! See http://struts.apache.org/missing-page

changes as #76

**Pull request merged**